variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,799 @@
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Variable Cost|Electricity|Biomass|Traditionnal:
+   description: variable cost of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: €/MWh
+
+Variable Cost|Electricity|Biomass|New and imported :
+   description: variable cost of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: €/MWh
+
+Variable Cost|Electricity|Nuclear:
+   description: variable cost of a typical nuclear power plant
+   unit: €/MWh
+
+Variable Cost|Electricity|Gas|OCGT:
+   description: variable cost of a typical Open Cycle Gas Turbine power plant
+   unit: €/MWh
+
+Variable Cost|Electricity|Gas|Coal:
+   description: variable cost of a typical coal power plant
+   unit: €/MWh
+
+Variable Cost|Electricity|Lignite:
+   description: variable cost of a typical Lignite power plant
+   unit: €/MWh
+
+# Variables related to thermal power plants
+Number of Units|Electricity|Biomass|Traditional:
+   description: number of units of kind "Traditionnal Biomass power plant "
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit:
+
+Number of Units|Electricity|Biomass|New and imported:
+   description: Number of units of kind "new biomass power plant "
+                biomass from dedicated energy crops and biomass imports.
+   unit:
+
+Number of Units|Electricity|Nuclear:
+   description: Number of units of kind "nuclear power plant "
+   unit:
+
+Number of Units|Electricity|Gas|OCGT:
+   description: Number of units of kind "Open Cycle Gas Turbine power plant"
+   unit:
+
+Number of Units|Electricity|Gas|Coal:
+   description: Number of units of kind "coal power plant "
+   unit:
+
+Number of Units|Electricity|Lignite:
+   description: Number of units of kind "Lignite power plant "
+   unit:
+Maximum Active power|Electricity|Biomass|Traditionnal:
+   description: Maximum active power of one typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: MW
+
+Maximum Active power|Electricity|Biomass|New and imported:
+   description: Maximum active power of one typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: MW
+
+Maximum Active power|Electricity|Nuclear:
+   description: Maximum active power of one typical nuclear power plant
+   unit: MW
+
+Maximum Active power|Electricity|Gas|OCGT:
+   description: Maximum active power of one typical Open Cycle Gas Turbine power plant
+   unit: MW
+
+Maximum Active power|Electricity|Gas|Coal:
```

Comment:
Is this a gas-fired power plant running on coal?

Reply:
@danielhuppmann I think this is the result of a (too) quick copy-paste.... should be Maximum Active power|Electricity|Coal

Reply:
No worries, I'm changing the files...and I'll be checking it.
I've found other ones...`Number of Units|Electricity|Gas|Coal`

Reply:
@erikfilias  same problem..... all gas|coal should be only coal.... sorry for that....